CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ set(HPX_LIBRARY_VERSION "${HPX_VERSION}")
 set(HPX_SOVERSION ${HPX_MAJOR_VERSION})
 set(HPX_PACKAGE_NAME HPX)
 
-if(MSVC)
+if(WIN32)
```

Comment:
This is only needed when generating solutions for visual studio. Is this the case?